plugins/scaffolder-backend/src/scaffolder/templater/cookiecutter.ts

```diff
@@ -16,17 +16,67 @@ import { TemplaterBase, TemplaterRunOptions } from '.';
  * limitations under the License.
  */
 import fs from 'fs-extra';
-
+import Docker from 'dockerode';
+import { JsonValue } from '@backstage/config';
+import { PassThrough } from 'stream';
 export class CookieCutter implements TemplaterBase {
+  private docker: Docker;
+  constructor() {
+    this.docker = new Docker();
+  }
+
+  private async fetchTemplateCookieCutter(
+    directory: string,
+  ): Promise<Record<string, JsonValue>> {
+    try {
+      return await fs.readJSON(`${directory}/cookiecutter.json`);
```

Comment:
When we will do overall scaffolder cleaning/refactoring we probably need to change it to path.resolve

Reply:
We might as well do some of this now maybe?

Reply:
Thinking that it would be good to implement a `create-react-app` templater too, which could also just spin up a docker container with `npx create-react-app` and have some values piped in.

Reply:
Sure, seems like a nice low-hanging fruit 👍